tuning/sft_trainer.py

```diff
@@ -175,7 +17,7 @@ def main(**kwargs):
         tune_config=prompt_tuning_config
     else:
         tune_config=None
-    train(model_args, data_args, training_args, tune_config)
+    TunedCausalLM.train(model_args, data_args, training_args, tune_config)
```

Comment:
this is the part in particular thats confusing calling it  TunedCausalLM.train

Reply:
Hey - I agree with you 😄 most of the confusion from here comes from trying to avoid naming collisions with stuff in Peft, transformers etc.

After discussing in person, I have split .train back out of this class and refactored the inference capabilities into a single script outside of the tuning package